src/js/components/Menu/__tests__/Menu-test.js

```diff
@@ -140,39 +140,36 @@ describe('Menu', () => {
         <Menu
           id="test-menu"
           label="Test"
-          items={[{ label: 'Item 1', onClick }, { label: 'Item 2' }]}
+          items={[{ label: 'Item 1' }, { label: 'Item 2', onClick }]}
         />
       </Grommet>,
     );
     expect(container.firstChild).toMatchSnapshot();
 
-    // pressing down 3x: first opens the drop,
-    // second moves to the first suggestion
-    // third moves to the last suggestion
+    // pressing space: opens drop
+    // tab moves to menu label
+    // second moves to first item
+    // third moves to second item
```

Comment:
This description no longer describes the current code flow, I think it should be adjusted.

Reply:
This description does refer to the expected flow, but if this is not what is happening please let me know. But this describes the test steps as:
1. space opens the menu
2. tab to the menu label
3. tab to the first item
4. tab to the second item
Then, the second item is selected with 'Enter'.